pkg/util/admission/testdata/elastic_cpu_work_queue

```diff
@@ -63,7 +63,7 @@ handle:      50ms
 admitted-work-done running=10ms allotted=50ms
 ----
 granter:    return-grant=40ms
-work-queue: 
+work-queue: adjustTenantUsed: tenantID=system additionalUsed=-40000000
```

Comment:
[minor nit] Could we format this in the time.Duration units like we do for the other parameters? (`running=`, `allotted=`, `return-grant=`). Perhaps avoid the snake casing too, formatting using something like:
```
work-queue: adjust-used: tenant=system [-/+]40ms
```